README.md

```diff
@@ -1,77 +1,80 @@
 
-### What's this all about?
+### Contributing
+To contribute, please review the Mozilla [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) and then visit our [how to contribute](docs/contributing.md) guide.
 
-This repository hosts the code and docs needed to integrate with the products offered by the Firefox
-[Application Services](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home)
-team.
+### Contact
+Get in touch with other community members on Matrix, the mailing list or through issues here on GitHub.
+- Matrix: [#sync:mozilla.org](https://chat.mozilla.org/#/room/#sync:mozilla.org) ([How to connect](https://wiki.mozilla.org/Matrix#Connect_to_Matrix))
+- Mailing list: application-services@mozilla.com
 
-If you are interested in getting involved in the development of those products
-then you're in the right place! Please review the more detailed guide on
-[how to contribute](docs/contributing.md) to this project as well as
-the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
+### Building the Rust Components
+1. Clone or Download the repository:
+```shell
+  git clone https://github.com/mozilla/application-services
+  ```
+2. Follow these instructions to install your [system-level dependencies](https://github.com/mozilla/application-services/blob/main/docs/build.md#building-application-services) (2 hrs)
+3. Fetch and install submodule dependencies
+```shell
+git submodule init
+git submodule update --recursive
+```
+4. Rust the a-s Rust unit tests
+```shell
+cargo test
+```
 
-If that's not why you're here, then instead you might enjoy:
+### Consumer build, integration and testing
+The application-services library primary consumers are Fenix (Firefox on Android) and Firefox iOS. Assure you are able to run integration tests (for Android and iOS if using MacOS) by following the instructions to build for Android and iOS integrations.  
 
-* The [Application Services Product
-  Portal](https://mozilla.github.io/application-services/), if you're looking to
-  use those products in your application.
-* The [Application Services Team Home](https://mana.mozilla.org/wiki/display/CLOUDSERVICES/Application+Services+Home)
-  on Mana, if you're trying to find out more about how we build them.
+#### Android integration builds and helpful tools
+* Build instructions to test [Fenix / android-components integration](https://github.com/mozilla/application-services/blob/main/docs/build.md#building-for-fenix) (2 hrs)
+* [Fenix Auto-publication workflow for android-components and application-services](https://github.com/mozilla-mobile/fenix/#auto-publication-workflow-for-android-components-and-application-services)
+* [Nimbus: using local servers for development](https://github.com/mozilla-mobile/fenix/#using-nimbus-servers-during-local-development)
```

Comment:
This nimbus one feels like it might be more appropriate in the nimbus component readme, once that's in the repo here.